Curtains/Curtains.Application/CurtainsService/EmailSender.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net;
+using System.Net.Mail;
+using Curtains.Application.CurtainsService.Interfaces;
+using Microsoft.Extensions.Configuration;
+
+namespace Curtains.Application.CurtainsService
+{
+    public class EmailSender : IMessageSender
+    {
+        private const string _smtpClient = "smtp.gmail.com";
+        private const int _port = 587;
+        private readonly IConfiguration _config;
+
+        public EmailSender(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        /// <summary>
+        /// Sends a message to a specific address to an email address
+        /// </summary>
+        /// <param name="customerEmail">The mail to which the message is sent</param>
+        public async Task SendAsync(string subject, string body, string customerEmail)
```

Comment:
By this name i can't understand what this method really do
Send Async what? where? why?

Reply:
renamed